main.nf

```diff
@@ -1131,112 +1131,119 @@ process busco_download_db {
 
 metabat_bins
     .transpose()
-    .combine(busco_db)
-    .set { metabat_db_busco }
+    .combine(ch_busco_db)
+    .set { ch_metabat_db_busco }
 
 /*
  * BUSCO: Quantitative measures for the assessment of genome assembly
  */
 process busco {
-    tag "${assembly}"
+    tag "${bin}"
     publishDir "${params.outdir}/GenomeBinning/QC/BUSCO/", mode: 'copy'
 
     input:
-    set val(assembler), val(sample), file(assembly), val(db_name), file(db) from metabat_db_busco
-
+    set val(assembler), val(sample), file(bin), val(db_name), file(db) from ch_metabat_db_busco
+ 
     output:
-    file("short_summary_${assembly}.txt") into (busco_summary_to_multiqc, busco_summary_to_plot)
-    val("$assembler-$sample") into busco_assembler_sample_to_plot
-    file("${assembly}_busco_log.txt")
-    file("${assembly}_buscos.faa")
-    file("${assembly}_buscos.fna")
+    set val(assembler), val(sample), file("short_summary.specific.${db}.${bin}.txt") into (ch_busco_multiqc, ch_busco_to_summary, ch_busco_plot)
+    file("${bin}_busco.log")
```

Comment:
At least this filename changed, probably the output.md needs an update too?

Reply:
The summary output and also the plots etc. are yet listed, I will add them. For "${assembly}_buscos.faa" just the variable name changed, not that actual file name.

Reply:
I meant `_busco_log.txt` -> `_busco.log`, and yes, thats nit-picky.